### PR TITLE
Allow -Xdump options for jcmd Dump.* commands

### LINKED ISSUE #655: Allow -Xdump options for jcmd Dump.* commands
**Issue or pull request number:**
https://github.com/eclipse/openj9/pull/10774

**Overview:**
jcmd Dump commands allow additional parameters to be specified to control the dumps. The defaults for system and heap dumps are modified to `request=exclusive+prepwalk`

**Release target:**
Eclipse OpenJ9 0.24.0

**Applies to the following JDK versions:**
All

**Applies to the following platforms:**
All

**For new command line options:**

jcmd Dump.system, Dump.heap, Dump.java, Dump.snap accept an optional `request=` parameter. Dump.heap accepts an optional `opts=` parameter. These parameters, including the `<file path>` parameter, can be in any order.

### PR DESCRIPTION
**Pull request by pmhayward**
Resolves #655

Signed-off-by: Peter Hayward <pmhayward@uk.ibm.com>

### CODE REVIEW

**Comment by pmhayward** [comment]
Jenkins doc stage

File: docs/version0.24.md [hunk 1]
```diff
@@ -71,6 +71,15 @@ The `-XX:[+|-]PortableSharedCache` option is now supported on IBM Z&reg; and POW
 The `-XX:[+|-]ShareAnonymousClasses` option currently enables and disables the storage of VM anonymous classes in the shared classes cache. From OpenJDK 15, this option now enables and disables the storage of hidden classes in the shared classes cache. See [`-XX:[+|-]ShareAnonymousClasses`](xxshareanonymousclasses.md) for more details about this option.
 
 
+### Additional parameters for `jcmd Dump` commands
+
+You can now include additional parameters for `jcmd Dump` commands to control both system and heap dumps. The new options are as follows:
```
Comment (pshipton) [inline docs/version0.24.md:76]: "to control both system and heap dumps" is incompatible with `Dump.system`, `Dump.heap`, `Dump.java`, `Dump.snap`, i.e. it doesn't mention java or snap dumps. Perhaps "to control both system and heap dumps" should just be removed.

`request=<parameter>` should refer to https://www.eclipse.org/openj9/docs/xdump/#requestltrequestsgt and `opts=<parameter>` to https://www.eclipse.org/openj9/docs/xdump/#optsltoptionsgt

**Comment by pmhayward** [comment]
Jenkins doc stage

File: docs/version0.24.md (hunk 1 above)
Comment (pmhayward) [reply docs/version0.24.md]: @pshipton ... updates made

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by pshipton** [comment]
Looks good, thanks.

**Review by pshipton — APPROVED** [review]

**Pull request merged**